gateway-discovery-cm/pom.xml

```diff
@@ -50,26 +45,16 @@
         <dependency>
             <groupId>com.cloudera.api.swagger</groupId>
             <artifactId>cloudera-manager-api-swagger</artifactId>
-            <version>${cloudera-manager-api-swagger.version}</version>  <!-- or CM version 6.0 and above -->
         </dependency>
         <dependency>
             <groupId>com.squareup.okhttp</groupId>
             <artifactId>okhttp</artifactId>
-            <version>${okhttp.version}</version>
         </dependency>
         <dependency>
             <groupId>org.apache.httpcomponents</groupId>
             <artifactId>httpcore</artifactId>
             <scope>compile</scope>
```

Comment:
Remove `<scope>compile</scope>` since that is the default and not needed. (I know its not directly related but its the same pom file)

Reply:
Done.